[Bug]: Fail to install pgvectorscale from source

### What happened?

When I build pgvectorscale by following [https://github.com/timescale/pgvectorscale/blob/main/DEVELOPMENT.md](https://github.com/timescale/pgvectorscale/blob/main/DEVELOPMENT.md), but error is thrown when I run the following command 
`cargo pgrx install --pg-config=/home/zijie/postgresql/pg17.2/bin/pg_config`. 
My clang version is 12.0.1.

### pgvectorscale extension affected

main

### PostgreSQL version used

17.2

### What operating system did you use?

centos8

### What installation method did you use?

Source

### What platform did you run on?

On prem/Self-hosted

### Relevant log output and stack trace

```bash
Generating bindings for pg17
  pg_config --configure CLANG = None
  Bindgen found clang version 12.0.1 (Red Hat 12.0.1-4.module_el8.5.0+1025+93159d6c)
  found libclang at /usr/lib64/libclang.so.12
  Found include dirs ["/usr/lib64/clang/12.0.1/include"]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/port.h:211:81: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/port.h:212:70: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/port.h:213:66: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/port.h:214:55: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/port.h:215:69: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/port.h:216:58: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/port.h:217:54: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/port.h:218:43: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/lib/stringinfo.h:172:67: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/lib/stringinfo.h:183:78: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:179:40: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:180:49: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:183:30: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:183:56: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:185:43: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:186:52: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:188:47: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:192:31: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:192:57: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:195:30: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:195:56: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:197:41: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:200:31: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:200:57: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:214:48: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:303:54: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/elog.h:538:47: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/palloc.h:148:44: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/zijie/postgresql/pg17.2/include/server/utils/palloc.h:149:80: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  Error: bindgen failed for pg17

  Caused by:
     0: failed to parse generated bindings
     1: expected identifier or `_`

  Location:
      /home/zijie/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/pgrx-bindgen-0.12.5/src/build.rs:751:40
warning: build failed, waiting for other jobs to finish...
```

### How can we reproduce the bug?

```bash

```

### Are you going to work on the bugfix?

None

--- a/pgvectorscale/Cargo.toml
+++ b/pgvectorscale/Cargo.toml
@@ -21,7 +21,7 @@ pg_test = []
 
 [dependencies]
 memoffset = "0.9.0"
-pgrx = "=0.12.5"
+pgrx = "=0.12.9"
 rkyv = { version = "0.7.43", features = ["validation"] }
 simdeez = { version = "1.0.8" }
 rand = { version = "0.8", features = ["small_rng"] }
@@ -31,8 +31,8 @@ serial_test = "3.2.0"
 once_cell = "1.20.1"
 
 [dev-dependencies]
-pgrx-tests = "=0.12.5"
-pgrx-pg-config = "=0.12.5"
+pgrx-tests = "=0.12.9"
+pgrx-pg-config = "=0.12.9"
 criterion = "0.5.1"
 tempfile = "3.15.0"
 

